spec/setup_test_database.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../lib/database_connection'
+
+def setup_test_database
+  conn = DatabaseConnection.setup('chitter_test')
+  conn.query("DROP TABLE peeps;")
```

Comment:
if you use "TRUNCATE peeps, user RESTART IDENTITY" you just delete the entries, not the columns and the id will restart"